packages/esm-patient-registration-app/src/patient-registration/section/patient-relationships/relationships-section.component.tsx

```diff
@@ -64,9 +64,7 @@ const RelationshipView: React.FC<RelationshipViewProps> = ({
   );
 
   const searchPerson = async (query: string) => {
-    const abortController = new AbortController();
```

Comment:
I'd keep the abortController here and pass it as an argument to fetch person (it shouldn't be here, but that's because it should be defined further up in the component).